### PR TITLE
SSUT-148: Create roroUnaccompanied page:

### PR DESCRIPTION
**Pull request by PerrettJ4**
- 3 fields
	- alphaNum, numString & varChar option validation
	- Length validation
	- Ferry company field optional
	- Navigation

### CODE REVIEW

File: app/viewmodels/checkAnswers/transport/RoroUnaccompaniedIdentitySummary.scala [hunk 1]
```diff
@@ -32,7 +32,10 @@ object RoroUnaccompaniedIdentitySummary {
     answers.get(RoroUnaccompaniedIdentityPage).map {
       answer =>
 
-      val value = HtmlFormat.escape(answer.field1).toString + "<br/>" + HtmlFormat.escape(answer.field2).toString
+        val value = {
```
Comment (AdamCoops) [inline app/viewmodels/checkAnswers/transport/RoroUnaccompaniedIdentitySummary.scala:35]: Do we need to include the optional ferry company here?

File: test/controllers/transport/RoroUnaccompaniedIdentityControllerSpec.scala [hunk 2]
```diff
@@ -35,21 +36,20 @@ import scala.concurrent.Future
 
 class RoroUnaccompaniedIdentityControllerSpec extends SpecBase with MockitoSugar {
 
-  val formProvider = new RoroUnaccompaniedIdentityFormProvider()
-  val form = formProvider()
+  private val formProvider = new RoroUnaccompaniedIdentityFormProvider()
+  private val form = formProvider()
+  private val alphaNum = "foobar123"
+  private val varChar = "f%%b&r.123"
```
Comment (giftig) [inline test/controllers/transport/RoroUnaccompaniedIdentityControllerSpec.scala:42]: These two vals aren't used

File: app/viewmodels/checkAnswers/transport/RoroUnaccompaniedIdentitySummary.scala (hunk 1 above)
Comment (giftig) [reply app/viewmodels/checkAnswers/transport/RoroUnaccompaniedIdentitySummary.scala:35]: This class is just for the CYA page and I think its content will probably change significantly when we complete that. For now a simple placeholder is ok.

File: test/controllers/transport/RoroUnaccompaniedIdentityControllerSpec.scala (hunk 2 above)
Comment (PerrettJ4) [reply test/controllers/transport/RoroUnaccompaniedIdentityControllerSpec.scala:42]: oh yeah, will amend now.

**Review by giftig — APPROVED** [review]

**Pull request merged**